src/containers/shared/utils.js

```diff
@@ -151,7 +151,10 @@ export const getLocalizedCurrencySymbol = (lang = 'en-US', currency = 'USD') =>
   return formatted.split('1')[0].trim();
 };
 
-export const formatLargeNumber = (d = 0, digits = 4) => {
+export const formatLargeNumber = (d = 0, digits = 6) => {
+  let variableDigits = 6;
+  let numberOfZeros = 0;
+  let numberCopy = d;
```

Comment:
Looks like you don't actually have to copy `d` - it's not modified anywhere else, so you can just use `d`